packages/ui/src/components/Modal/modal.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { Modal as DefaultModal, ModalProps as DefaultModalProps } from 'antd'
+import classNames from 'classnames'
+import { Typography } from '../Typography'
+import { Cross } from './cross'
+
+enum CondoModalWidth {
+    small = 570,
+    big = 900,
+    fitContent = 'fit-content',
+}
+
+type CondoModalProps = {
+    title: string
+    open: boolean
+    width?: CondoModalWidth
+}
+
+export type ModalProps = Pick<DefaultModalProps,
+'children'
+| 'footer'
+| 'className'
+| 'afterClose'
+| 'maskClosable'
+| 'destroyOnClose'
+| 'getContainer'
+| 'zIndex'
+> & CondoModalProps
+
+const MODAL_CLASS_PREFIX = 'condo-modal'
+
+const Modal: React.FC<ModalProps> = (props) => {
+    // NOTE The popup has three widths ('small', 'big', 'auto')
+    // But it also remains possible to set a custom value
```

Comment:
It should not be possible. Let's just set it to only 3 values

Reply:
oops, this note should be removed)